bitnami/contour/README.md

```diff
@@ -65,10 +65,11 @@ The following tables lists the configurable parameters of the contour chart and
 | `contour.customResourceDeletePolicy`               | Deletion hook of customResources viah helm hooks (only helm v2)                                        | `nil`                                                   |
 | `contour.nodeSelector`                             | Node labels for contour pod assignment                                                                 | `{}`                                                    |
 | `contour.tolerations`                              | Tolerations for contour pod assignment                                                                 | `[]`                                                    |
+| `contour.antiAffinity`                             | Contour anti-affinity policy (`soft`, `hard` or `""`)                                                  | `soft`                                                  |
```

Comment:
don't you think we should be more specific here? `contour.antiAffinityPolicy`

Reply:
I copied this from the influxdb chart (https://github.com/bitnami/charts/blob/master/bitnami/influxdb/values.yaml#L157).
But yes I also think we should be more specific